R/human_infection.R

```diff
@@ -61,20 +61,25 @@ calculate_infections <- function(
     infection_outcome
 ) {
   
-  if(bitten_humans$bitten_humans$size() == 0){return(bitten_humans$bitten_humans)}
-  
   if(parameters$parasite == "falciparum"){
-    source_humans <- variables$state$get_index_of(c('S','A','U'))$and(bitten_humans$bitten_humans)
+    
+    if(bitten_humans$size() == 0){return(bitten_humans)}
```

Comment:
This line isn't clear to me, can't we just wrap this section in a more classic if statement like:
```
if(bitten_humans$size() > 0){
Do this stuff
}
```